boilerplate/openshift/golang-osd-operator/codecov.sh

```diff
@@ -9,7 +9,11 @@ CI_SERVER_URL=https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test
 COVER_PROFILE=${COVER_PROFILE:-coverage.out}
 JOB_TYPE=${JOB_TYPE:-"local"}
 
-make -C "${REPO_ROOT}" go-test TESTOPTS="-coverprofile=${COVER_PROFILE}.tmp -covermode=atomic -coverpkg=./..."
+# `-p 4`: Set concurrency to four threads. By default it's the number of
+# procs, which seems to be 16 in the CI env. Some consumers' coverage
+# jobs were # regularly getting OOM-killed; so do this rather than boost
+# the pod resources unreasonably.
+make -C "${REPO_ROOT}" go-test TESTOPTS="-coverprofile=${COVER_PROFILE}.tmp -covermode=atomic -coverpkg=./... -p 4"
```

Comment:
Can we make this a variable? Maybe COV_THREAD_COUNT?

Reply:
Good idea. Done.